tests/unit/debounce-task-test.js

```diff
@@ -46,6 +46,15 @@ module('ember-lifeline/debounce-task', function(hooks) {
     }, 10);
   });
 
+
+  test('debounceTask triggers an assertion when delay argument is not a number or not passed', function(assert) {
+    this.obj = this.getComponent({ doStuff() {} });
+
+    assert.throws(() => {
+      debounceTask(this.obj, 'doStuff', 'bad');
```

Comment:
Can we add a test for the specific case you found? Specifically, when an object is incorrectly passed?